api-contracts/openapi/openapi.yaml

```diff
@@ -170,7 +174,7 @@ paths:
     $ref: "./paths/workflow/workflow.yaml#/workflowRunsMetrics"
   /api/v1/tenants/{tenant}/workflow-runs/{workflow-run}:
     $ref: "./paths/workflow/workflow.yaml#/workflowRun"
-  /api/v1/tenants/{tenant}/workflow-runs/{workflow-run}/shape:
+  /api/v1/tenants/{tenant}/workflow-runs/{workflow-version-id}/shape:
```

Comment:
Is it intentional that we're changing a `/api/v1` endpoint here?

Reply:
nice catch, I'll unwind this, the change here is unused too